sklearn/semi_supervised/label_propagation.py

```diff
@@ -32,7 +32,7 @@
 --------
 >>> from sklearn import datasets
 >>> from sklearn.semi_supervised import LabelPropagation
->>> label_prop_model = LabelPropagation()
+>>> label_prop_model = LabelPropagation(max_iter=1000)
```

Comment:
Is the default max_iter something we should by changing while we are making other backwards incompatible changes? Or is the current default reasonable